IFC4x3/Constants/n/NAVIGATIONALCHANNEL_3D1nsuCyT7vvYNgvIKdsJ$.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NAVIGATIONALCHANNEL_3D1nsuCyT7vvYNgvIKdsJ" Name="NAVIGATIONALCHANNEL" UniqueId="cd071db8-33c7-47e7-9897-ab94949f64ff">
-	<Documentation>A natural navigable watercourse (such as river) that needs to be managed or have improvements applied. this also include defined navigational areas in open seas and bays.
+	<Documentation>A natural navigable watercourse (such as a river) that needs to be managed or have improvements applied. this also include defined navigational areas in open seas and bays.
```

Comment:
Capitalise 'T' in 'this' and 'include' ->'includes'.

Reply:
sorted